mypy_django_plugin/transformers/models.py

```diff
@@ -273,14 +273,17 @@ def run_with_model_cls(self, model_cls: Type[Model]) -> None:
             if isinstance(relation, (ManyToOneRel, ManyToManyRel)):
                 try:
                     related_manager_info = self.lookup_typeinfo_or_incomplete_defn_error(fullnames.RELATED_MANAGER_CLASS)  # noqa: E501
-                    if 'objects' not in related_model_info.names:
-                        raise helpers.IncompleteDefnException()
                 except helpers.IncompleteDefnException as exc:
                     if not self.api.final_iteration:
                         raise exc
                     else:
                         continue
 
+                if 'objects' not in related_model_info.names:
```

Comment:
Thanks! I would be also glad to see a test case for this change.

Can you please provide one that was failing for you?

Reply:
I’ll try to recreate it again, but i have been unsuccessful so far. I’ll try removing code from our actual project until i have a reproduce instead of creating a test case from the ground up.

Reply:
thanks!